app/components/Comments/CommentDisplay.jsx

```diff
@@ -143,6 +144,22 @@ export class CommentDisplay extends React.PureComponent {
     </React.Fragment>
   }
 
+  renderUserHeader(user, withoutActions) {
```

Comment:
personal tastes here : I dislike this kind of `if` use, specially when it's avoidable 
why not 
```Javascript
renderUserHeader(user, withoutActions) {
  user ? (
    <span>
      <UserPicture user={user} size={USER_PICTURE_SMALL}/>
      <UserAppellation user={user} withoutActions={withoutActions}/>
    </span>
  ) : (
    <span className="anonymous">
      {this.props.t('anonymous')}
    </span>
  )
}
```

Reply:
forgot `return` keyword
editing snippet

Reply:
Done